src/connection.rs

```diff
@@ -74,7 +74,63 @@ impl Connection {
     ///
     /// # Example
     /// ```
-    /// todo!()
+    /// use zbus::{fdo::DBusProxy, MessageType, MatchRule, Address};
+    /// use futures_lite::future::{block_on, race, yield_now};
+    /// use futures_lite::pin;
+    /// use futures_lite::StreamExt;
+    /// let receive_good_event = async {
+    ///    let connection = atspi::Connection::open().await.expect("Could not open a11y bus.");
+    ///    let object_match_rule = MatchRule::builder()
+    ///        .msg_type(MessageType::Signal)
+    ///        .interface("org.a11y.atspi.Event.Object").expect("Can not build MatchRule with interface org.a11y.atspi.Event.Object")
+    ///        .build();
```

Comment:
Maybe just `unwrap()` in examples. I have recently [come to learn (std lib docs)](https://doc.rust-lang.org/stable/std/result/enum.Result.html#recommended-message-style) that the  `expect()` message should say why we expect the Result<> should be Ok. (why is it justified to call expect in this situation)
There generally is too little context or relevance for that in an example, plus, a simple unwrap is shorter.

Reply:
I can change it to unwrap.